src/tolerances.jl

```diff
@@ -3,7 +3,7 @@
 # ------------------------------------------------------------------
 
 const ATOL64 = ScopedValue(1e-10)
```

Comment:
```suggestion
const ATOL64 = ScopedValue(1.0e-10)
```
Why does the formatter want the additional zero for the `Float32` version, but not for the `Float64` version?

Reply:
Don't know. Thanks for catching this.